backstop/scenarios/header.js

```diff
@@ -4,27 +4,32 @@ module.exports = [
   // Header types
   scenario('Type Default', {
     selectors: ['#demo-default'],
+    misMatchThreshold: 0.02,
   }),
   scenario('Type Slim', {
     selectors: ['#demo-slim'],
+    misMatchThreshold: 0.02,
   }),
 
   // Hover & focus states
   //// Default
   scenario('Type Default State hover & focus', {
     actions: [hover('#test-button-1'), focus('#test-button-2')],
     selectors: ['#demo-default'],
+    misMatchThreshold: 0.02,
   }),
 
   //// Slim
   scenario('Type Slim State hover & focus', {
     actions: [hover('#test-button-3'), focus('#test-button-4')],
     selectors: ['#demo-slim'],
+    misMatchThreshold: 0.02,
   }),
 
   //// Split
   scenario('Type Split State hover & focus', {
     actions: [hover('#test-button-6'), focus('#test-button-6')],
     selectors: ['#demo-split'],
+    misMatchThreshold: 0.05,
```

Comment:
I think this will be Chrome again rendering images a bit different every time. Because previously we had the same problem with `Tile` on hover where images are scaled up. I then made screenshot, refreshed, made another screenshot and compared them with image comparer. Turns out images always differs a bit in few pixels.